tmt/steps/__init__.py

```diff
@@ -683,11 +683,13 @@ def wake(self) -> None:
         # Special handling for the report step to always enable force mode in
         # order to cover a very frequent use case 'tmt run --last report'
         # FIXME find a better way how to enable always-force per plugin
-        if (isinstance(self, tmt.steps.report.Report) and
-                self.data[0].how in ['display', 'html']):
-            self.debug("Report step always force mode enabled.")
-            self._workdir_cleanup()
-            self.status('todo')
+        if isinstance(self, tmt.steps.report.Report):
+            self._assert_default_phase()
```

Comment:
Which commands / scenario would actually trigger this code branch? Seems that checks around line 885 always ensure that there are some raw data and they run before `wake()`. Is it included "just to be sure"?

Reply:
True, `_assert_default_phase()` may no longer be necessary. I will try it out.